deps/wazuh_testing/wazuh_testing/qa_docs/lib/sanity.py

```diff
@@ -55,7 +59,7 @@ def validate_fields(self, required_fields, available_fields):
             for field in required_fields:
                 if not check_existance(available_fields, field):
                     self.add_report(f"Mandatory field '{field}' is missing in file {self.scan_file}")
-                    logging.error(f"Mandatory field '{field}' is missing in file {self.scan_file}")
+                    Sanity.LOGGER.error(f"Mandatory field '{field}' is missing in file {self.scan_file}")
```

Comment:
```suggestion
                    Sanity.LOGGER.error(f"Mandatory field '{field}' is missing in the file {self.scan_file}")
```

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/86e5045beb49f8e91dbb42b6e59f8d3164a57523

Reply:
Done in: https://github.com/wazuh/wazuh-qa/commit/9e21347fdd74c17953f57a11cdf67be1d85b336f